### PR TITLE
MULTISITE-19240: Fixes first round for toolkit

### PR DESCRIPTION
**Pull request by ghost**
@jonhy81 , I need some feedback on this solution. Instead of loading build properties manually I emulate a phing project. This is ugly but I think it's the only solution to make it work again.

### CODE REVIEW

File: src/Console/Command/CheckCodingStandardsCommand.php [hunk 1]
```diff
@@ -48,13 +48,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
-        $standard = !empty($input->getOption('standard')) ? $input->getOption('standard') : $properties['phpcs-config'];
-        $basedir = $input->getOption('basedir');
+        $standard = !empty($input->getOption('standard')) ? $input->getOption('standard') : $properties['phpcs.config'];
+        $basedir = $input->getOption('project.basedir');
 
         //$width = !empty($input->getOption('width')) ? $input->getOption('width') : 80;
         $show = $input->getOption('show') ? true : false;
         ob_start();
-        $executable = $basedir . "/ssk/phpcs";
+        $executable = $basedir . "/toolkit/phpcs";
```
Comment (jonhy81) [inline src/Console/Command/CheckCodingStandardsCommand.php]: ```${toolkit.dir.bin}/phpcs``` instead, maybe more flexible

File: src/Console/Command/ScanCommentedCodeCommand.php [hunk 2]
```diff
@@ -36,23 +36,23 @@ protected function configure()
...
         $dirname = !empty($input->getOption('directory')) ? $input->getOption('directory') : getcwd();
         // @codingStandardsIgnoreLine
         $exclude_dirs = !empty($input->getOption('exclude-dirs')) ? '--ignore=' . $input->getOption('exclude-dirs') . ' ' : '';
-        $basedir = !empty($input->getOption('basedir')) ? $input->getOption('basedir') : $properties['basedir'];
+        $basedir = !empty($input->getOption('project.basedir')) ? $input->getOption('project.basedir') : $properties['project.basedir'];
         //$width = !empty($input->getOption('width')) ? $input->getOption('width') : 80;
         $show = $input->getOption('show') ? true : false;
         ob_start();
-        $executable = $basedir . "/ssk/phpcs";
+        $executable = $basedir . "/toolkit/phpcs";
```
Comment (jonhy81) [inline src/Console/Command/ScanCommentedCodeCommand.php]: Maybe ```${toolkit.dir.bin}/phpcs``` can simplify and remove the hardcoded `toolkit` reference

File: src/Console/Helper/PhingPropertiesHelper.php [hunk 3]
```diff
@@ -188,16 +193,26 @@ private function getAllSettings($buildfile = '')
             $buildfile = $this->findPhingBuildFile();
         }
         if ($buildfile) {
+            \Phing::startup();
```
Comment (jonhy81) [inline src/Console/Helper/PhingPropertiesHelper.php]: Looks fine to me.

File: src/Console/Command/CheckCodingStandardsCommand.php (hunk 1 above)
Comment (ghost) [reply src/Console/Command/CheckCodingStandardsCommand.php]: Good idea, will see if I can do that. Need to simplify the property loading anyway. Load once before qa execution and make sure all commands have access to them.

File: src/Console/Command/ScanCommentedCodeCommand.php (hunk 2 above)
Comment (ghost) [reply src/Console/Command/ScanCommentedCodeCommand.php]: 👍

File: src/Console/Helper/PhingPropertiesHelper.php (hunk 3 above)
Comment (ghost) [reply src/Console/Helper/PhingPropertiesHelper.php]: It's ugly because phing uses includes instead of PSR4 :(

**Pull request merged**